### PR TITLE
Feature/create button component

### PR DESCRIPTION
**Pull request by CBinh710**
## Create button component
https://github.com/users/CBinh710/projects/2/views/1?pane=issue&itemId=72043396
image

### CODE REVIEW

File: practice-one/src/styles/components/button.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.button {
+  background-color: var(--color-primary);
+  border: none;
+  border-radius: 100px;
+  width: 162px;
+  height: 45px;
```
Comment (hungdo-agilityio) [inline practice-one/src/styles/components/button.css]: Should not set fixed width, height

File: practice-one/package-lock.json
Comment (hungdo-agilityio) [inline practice-one/package-lock.json:1]: Why this file change

File: practice-one/src/components/header.html [hunk 2]
```diff
@@ -9,6 +9,8 @@
             <li><a href="#">Our Menu</a></li>
             <li><a href="#">Our Popular Food</a></li>
         </ul>
-        <button>Download</button>
+        <include src="./components/button.html" locals='{
+          "text": "Download"
+      }'></include>
```
Comment (hungdo-agilityio) [inline practice-one/src/components/header.html]: Recheck the indent here should be align with open tag `include`

File: practice-one/src/styles/components/button.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  color: var(--color-light);
+  font-family: var(--font-family-normal);
```
Comment (hungdo-agilityio) [inline practice-one/src/styles/components/button.css]: If this font family is primarily used for the whole app, then set font family for `html` then
```
html, body {
  font-family: var(--font-family-primary); // Rename this into font-family-primary
}
```

File: practice-one/src/styles/components/button.css (hunk 1 above)
Comment (CBinh710) [reply practice-one/src/styles/components/button.css]: thanks , i understood and fixed it

File: practice-one/package-lock.json
Comment (CBinh710) [reply practice-one/package-lock.json:1]: I have some issues when using posthtml-include nested so i try many ways to fix it ,maybe on these process it was added

File: practice-one/src/styles/components/button.css (hunk 3 above)
Comment (CBinh710) [reply practice-one/src/styles/components/button.css]: I understood, thanks very much

File: practice-one/src/components/header.html (hunk 2 above)
Comment (CBinh710) [reply practice-one/src/components/header.html]: i was align it , thanks

File: practice-one/package-lock.json
Comment (hungdo-agilityio) [reply practice-one/package-lock.json:1]: @CBinh710 Bypass this for now, but always remember different version of packages can cause unknown issues, so we need to investigate when package version change

File: practice-one/src/components/header.html [hunk 4]
```diff
@@ -9,6 +9,8 @@
             <li><a href="#">Our Menu</a></li>
             <li><a href="#">Our Popular Food</a></li>
         </ul>
-        <button>Download</button>
+        <include src="./components/button.html" locals='{
+        "text": "Download"
```
Comment (hungdo-agilityio) [inline practice-one/src/components/header.html]: Correct this indent, this should be 1 level lower than `include` tag

**Review by hungdo-agilityio — APPROVED** [review]

File: practice-one/src/components/header.html (hunk 4 above)
Comment (CBinh710) [reply practice-one/src/components/header.html]: thanks

**Pull request merged**